packages/wrangler/src/assets.ts

```diff
@@ -340,7 +328,7 @@ function logAssetUpload(line: string, diffCount: number) {
 			"   (truncating changed assets log, set `WRANGLER_LOG=debug` environment variable to see full diff)";
 		logger.info(chalk.dim(msg));
 	}
-	return diffCount++;
+	return ++diffCount;
```

Comment:
This was never incrementing and thus would print out 100k lines, fixed the limiting